src/firebolt/model/V2/engine.py

```diff
@@ -60,14 +60,22 @@ class Engine(FireboltBaseModel):
         "WARMUP",
         "ENGINE_TYPE",
     )
-    DROP_SQL: ClassVar[str] = "DROP ENGINE {}"
+    ALTER_PARAMETER_NAMES_V2: ClassVar[Tuple] = (
+        "NODES",
+        "TYPE",
+        "AUTO_STOP",
+        "RENAME_TO",
+        "",
+        "",
```

Comment:
Why do we have empty strings here?

Reply:
Fair enough, not feeling too strongly about it. Maybe let's just add a comment that this is left for compatibility.